test/Nancy.Tests/Unit/Security/CsrfFixture.cs

```diff
@@ -34,8 +35,10 @@ public CsrfFixture()
             this.pipelines = new MockPipelines();
 
             this.cryptographyConfiguration = CryptographyConfiguration.Default;
-
-            this.objectSerializer = new DefaultObjectSerializer();
+            var fakeAssCatalog = A.Fake<IAssemblyCatalog>();

```

Comment:
:smile: This is the same as the `adq` variable, why shorten it? `fakeAssemblyCatalog` is waaaaay more descriptive. :+1: for changing this, even if it's not for the same reason @phillip-haydon pointed out ;)